src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.tsx

```diff
@@ -46,7 +46,7 @@ export type DefaultTranscriptListItemProps = {
   toggleTranscriptInfo: (id: string) => void;
 };
 
-// NOTE: the width of the middle column is the same as the width of GeneOverviewImage, i.e. 695px
+// NOTE: the width of the middle column is the same as the width of GeneOverviewImage, i.e. $gene_image_width in EV constants
```

Comment:
We can probably safely delete this comment; it doesn't seem to be communicating anything useful.